modules/bullet/3.26.0-rc0/MODULE.bazel

```diff
@@ -0,0 +1,11 @@
+module(
+    name = "bullet",
+    version = "3.26.0-rc0",
+    compatibility_level = 3,
+)
+
+bazel_dep(name = "bazel_skylib", version = "1.7.1")
+bazel_dep(name = "eigen", version = "3.4.0.bcr.3")
```

Comment:
Are `eigen` and `tinyxml2` actually used by this module? I'm not seeing them in the BUILD file.

Reply:
actually not needed. I removed them.